### PR TITLE
Camden market case study

### PR DESCRIPTION
**Pull request by cwilgenhoff**
### Motivation

- Candem market case study

### Test plan

- TODO: Write a test plan.

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

File: site/compiler/people.test.js [hunk 1]
```diff
@@ -37,10 +37,13 @@ describe('site/compiler', () => {
...
-      expect(routes[11].filePath).to.equal('about-us/people/index.html');
-      expect(routes[12].filePath).to.equal('about-us/people/category/engineering/index.html');
-      expect(routes[13].filePath).to.equal('about-us/people/category/leadership/index.html');
+      const filePaths = routes.map(r => r.filePath);
+
+      expect(filePaths).to.include.members([
+        'about-us/people/index.html',
+        'about-us/people/category/engineering/index.html',
+        'about-us/people/category/leadership/index.html',
+      ]);
```
Comment (lazydevorg) [inline site/compiler/people.test.js]: Happiness
Happiness

File: site/compiler/people.test.js [hunk 2]
```diff
@@ -142,14 +151,15 @@ describe('site/compiler', () => {
...
+
+        expect(filePaths).to.include.members([
+          'about-us/people/index.html',
+          'about-us/people/category/everyone/page-2/index.html',
+          'about-us/people/category/engineering/index.html',
           'about-us/people/category/engineering/page-2/index.html',
-        );
-        expect(routes[15].filePath).to.equal('about-us/people/category/leadership/index.html');
+          'about-us/people/category/leadership/index.html',
+        ]);
```
Comment (lazydevorg) [inline site/compiler/people.test.js]: so much thanks for that! 🎉

**Comment by gregdorward** [comment]
Hey @cwilgenhoff, there's a spelling mistake in the url, could you change /our-work/case-study/candem/ to /our-work/case-study/camden/ please?

**Comment by chris-brookes** [comment]
@cwilgenhoff - that spelling mistake is in a number of other places too:

screen shot 2017-07-12 at 11 59 06

File: site/compiler/test.js [hunk 3]
```diff
@@ -20,19 +20,23 @@ describe('site/compiler', () => {
...
+
+      expect(filePaths).to.include.members([
+        'index.html',
+        'what-we-do/index.html',
+        'our-work/index.html',
+        'about-us/index.html',
+        'about-us/join-us/index.html',
+        'about-us/events/index.html',
+        'our-work/case-study/retailer/index.html',
+        'our-work/case-study/candem/index.html',
```
Comment (gregdorward) [inline site/compiler/test.js]: This needs to be changed to 'Camden'. I think it needs to be a bulk update, as this spelling's used elsewhere as well

**Comment by cwilgenhoff** [comment]
Bad typo. It should be fixed by now.

**Comment by chris-brookes** [comment]
@cwilgenhoff -
1. The image in the header is going too far to the right. It should not pass the right hand side of the main content.

screen shot 2017-07-13 at 11 25 21


2. Now there is no background image in the header of tablet and mobile layouts, the main header image should be in the center of the page.

screen shot 2017-07-13 at 11 42 53

3. The page should not show 3 slides on all views. I think you should remove the changes to the WhatToReadNext from this PR and we should do all the work in that area in #490

**Comment by gregdorward** [comment]
Hey @martinhartt, would you mind taking a look at this one when you get a chance to make the following change please?

Update the font size of the following element to 24 - ._fonts__fontL___349eg, ._fonts__fontM___2uO71, ._fonts__fontStaticM___1IwFM

**Comment by gregdorward** [comment]
screen shot 2017-07-17 at 11 14 01

File: site/pages/our-work/case-study/camden-market/style.css [hunk 4]
```diff
@@ -84,19 +84,20 @@
 }
 
 .header__container {
-  max-width: 760px;
-  margin: 0 0 auto auto;
+  max-width: 1160px;
+  margin: auto;
   position: relative;
-  width: 100%;
+  width: calc(100% - 120px);
```
Comment (martinhartt) [inline site/pages/our-work/case-study/camden-market/style.css]: Nice spot, this explains why there wasn't a correct margin for certain screen sizes

**Comment by chris-brookes** [comment]
With the font size change for mobile (commit 25f3a94), the font was also changed for the tablet view. It should be kept at 30px for tablet. 
Desktop - 40px
Tablet - 30px
Mobile - 24px

**Comment by chris-brookes** [comment]
Font changes look fine on the Camden and Retailer case study page.

**Pull request merged**